ChangeLog

```diff
@@ -80,6 +80,11 @@ modules are added.
 * Make ``using-constant-test`` detect constant tests consisting of list literals like ``[]`` and
   ``[1, 2, 3]``.
 
+* New checker ``unused-protected-member``. Emitted when a protected member (i.e., starts with ``_``) of a class
```

Comment:
It's OK situation (not a bug). Typically such methods defined for calling in children classes.

Reply:
yep, I realise this -- which was why i closed this PR.

Correct implementation here #4504